### ISSUE TITLE
Produce Imagemagick Script From Operations

### ISSUE THREAD

**Issue by austinliuigi** [TeaM-TL/FotoKilof#167]
**Is your feature request related to a problem? Please describe.**
It would be useful if you could save the operations you make on an image to a script that reproduces them.

**Comment by TeaM-TL** [comment]
you are right, I though about it: https://github.com/TeaM-TL/FotoKilof/issues/69
To not duplicate, I will close this issue. If you have an idea, please continue on issues mentioned above
